priv/repo/migrations/20220829103340_add_workflows_table.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Lightning.Repo.Migrations.AddWorkflowsTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:workflows, primary_key: false) do
+      add :id, :binary_id, primary_key: true
+      add :name, :string
```

Comment:
i think we need a `project_id` field and then a unique constraint on `project_id + name`

Reply:
Like a `belongs to` relationship between `workflows` and `projects` ?